geoviews/util.py

```diff
@@ -99,14 +99,15 @@ def project_extents(extents, src_proj, dest_proj, tol=1e-6):
             geom_in_src_proj = geom_clipped_to_dest_proj
         try:
             geom_in_crs = dest_proj.project_geometry(geom_in_src_proj, src_proj)
-        except ValueError:
-            src_name =type(src_proj).__name__
-            dest_name =type(dest_proj).__name__
-            raise ValueError('Could not project data from %s projection '
-                             'to %s projection. Ensure the coordinate '
-                             'reference system (crs) matches your data '
-                             'and the kdims.' %
-                             (src_name, dest_name))
+        except (ValueError, IndexError):
+            src_name = type(src_proj).__name__
+            dest_name = type(dest_proj).__name__
+            raise ValueError(
+                f'Could not project data from {src_name} projection '
+                f'to {dest_name} projection. Ensure the coordinate '
+                'reference system (CRS) matches your data and the kdims, '
+                'and the data is not ouside the bounds of the CRS.'
+            ) from None
```

Comment:
Given that this is a viz tool designed to support data exploration, isn't there an argument for simply warning about data that can't be projected into these coordinates, while projecting the remainder of the data? E.g. if people have data extending to the poles and use a projection not defined for the full extent, seems like they should be able to see the rest of the data anyway. Otherwise they'll have to figure out the extent of this projection and figure out how to clip out that data, right?

Reply:
This is actually what it already does. The IndexError is happening because there is no data after the conversion. I wanted to show a small example of it happening, but I could not get it to work without changing the code. See comment below.